dexs/mach/deployments.ts

```diff
@@ -0,0 +1,76 @@
+import { CHAIN } from "../../helpers/chains";
+
+type Deployment = {
+    address: string,
+    start: number,
+}
+
+const DEPLOYMENTS: { [chain: string]: Deployment } = {
+    [CHAIN.ETHEREUM]: {
+        address: "0x5d8bca5F0b3D9c9513a75D0206dAF0b4FF8bda95",
+        start: 20299258,
```

Comment:
thanks for the PR, the code is mostly good except for this part, this needs to unix timestamp or string like `'2023-02-03'`

Reply:
also, removed the env file, think you were leaking your api key